CesiumGeometry/include/CesiumGeometry/Axes.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "CesiumGeometry/Library.h"
+#include <glm/mat3x3.hpp>
+
+namespace CesiumGeometry {
+
+/**
+ * @brief Coordinate system conversion matrices
+ */
+struct CESIUMGEOMETRY_API Axes final {
```

Comment:
it just a small nick pick from me. I think maybe `AxisTransform` is a better name? cause Axes is kind of similar to `Axis.h` header

Reply:
Yeah, I often hesitate with stuff like that. 

On the one hand, it was supposed to "mimic" the https://github.com/CesiumGS/cesium/blob/master/Source/Scene/Axis.js . On the other hand, there are language-specific best practices. (E.g. in the Java world, a class that only contains static members/methods is indicated by the Plural, e.g. when there is a class `Person`, then `Persons` contains the static utility methods). 

I went for `AxisTransforms` now, roughly aligned with https://github.com/CesiumGS/cesium/blob/master/Source/Core/Transforms.js (but specialized to transforming the up-axis). 

An aside: We now have the enum `Axis.X`, but I could imagine that having some

    const glm::dvec4 X_AXIS{ 1.0, 0.0, 0.0, 0.0 };

somehere could also make sense sooner or later. (Or should it be a `dvec3` then? ... Well ....)